backend/bin/server.js

```diff
@@ -149,7 +149,7 @@ app.get('/api/v3/creator/me', async (req, res) => await iowrapper('creatorGet',
 
 /* below, the new API for advertising */
 app.get('/api/v2/ad/video/:videoId', async (req, res) => await iowrapper('adsPerVideo', req, res));
-app.get('/api/v2/ad/channel/:channelId', async (req, res) => await iowrapper('adsPerChannel', req, res));
+app.get('/api/v2/ad/channel/:channelId/since-:sdate/till-:edate', async (req, res) => await iowrapper('adsPerChannel', req, res));
```

Comment:
Isn't better to move this variables to `query` params?
It will make it also more REST friendly